draft-irtf-cfrg-vdaf.md

```diff
@@ -5098,10 +5098,40 @@ possible. See {{xof-vs-ro}} for more information.
 | FieldLeaf  | `Field255` ({{fields}}) |
 {: #idpf-bbcggi21-param title="Constants and type definitions for the IDPF of BBCGGI21."}
 
-### Key Generation
+### Overview
+
+On a high level, the IDPF maps a key generator's input `(alpha, beta_1, ..., beta_BITS)` onto
+a binary tree with `BITS+1` levels, where each edge going from a parent node to a left child is
+labeled `0`, and each right edge is labeled `1`. Then each leaf node corresponds to a value
+`x` in `0, ..., 2^BITS`, and the labels on the path from the root to `x` contain the binary
+decomposition of `x`, starting with the most significant bit. Finally, all inner nodes have
```

Comment:
Following #433, we may want to talk about these as bit strings, rather than integers, for consistency.

Reply:
Fixed.